rpcs3/rpcs3qt/gui_settings.cpp

```diff
@@ -235,15 +236,16 @@ void gui_settings::BackupSettingsToTarget(const QString& friendlyName)
 
 QStringList gui_settings::GetStylesheetEntries()
 {
-	QStringList nameFilter;
-	nameFilter << "*.qss";
-	QFileInfoList entries = m_settingsDir.entryInfoList(nameFilter, QDir::Files);
-	QStringList res;
-	for (const QFileInfo &entry : entries)
-	{
-		res.append(entry.baseName());
-	}
-
+	QStringList nameFilter = QStringList("*.qss");
+	QStringList res = gui::utils::get_dir_entries(m_settingsDir, nameFilter);
+#ifdef __linux__
+	// Makes stylesheets load if using AppImage or installed to /usr/bin
+	QDir linuxStylesheetDir = QCoreApplication::applicationDirPath() + "/../share/rpcs3/GuiConfigs/";
+    res.append(gui::utils::get_dir_entries(linuxStylesheetDir, nameFilter));
+    res.removeDuplicates();
+#endif
+    res.sort(Qt::CaseInsensitive);
+    
```

Comment:
are these 4 lines of space indentations (244-247) the unwanted abomination I was talking about?

is this april fools?

Reply:
I'm not seeing them. Maybe just April Fools on your end?

Reply:
oh wait, nvm, I dunno what happened there.